### PR TITLE
creates searchbutton component

### LINKED ISSUE #23: Create search button
The search button:
- has a search icon
- has `hovered` state
- has `pressed` state

### PR DESCRIPTION
**Pull request by arinacimpac**
attempt at creating the search button component

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,6 +3,9 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@fortawesome/fontawesome-svg-core": "^1.2.36",
+    "@fortawesome/free-solid-svg-icons": "^5.15.4",
+    "@fortawesome/react-fontawesome": "^0.1.15",
```
Comment (ileahu-ellation) [inline package.json]: Let's remove these packages

File: src/components/Searchbutton/Searchbutton.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+const Searchbutton = ({className}) => (
+        <button className={classNames(className,styles.Iconstyle)}> 
+
+        <SearchSVG/>
+        </button>
+    )
+Searchbutton.propTypes = {
+    className: propTypes.string,
+
+}
```
Comment (ileahu-ellation) [inline src/components/Searchbutton/Searchbutton.js:16]: Can you check if Prettier is working for you? It seems that it doesn't format these files

File: src/App.js [hunk 3]
```diff
@@ -1,29 +1,42 @@
...
+
+
+
+function App() {
+  return (
+    
+    
+
+    <StoreProvider>
+      <div className={styles.app}>
```
Comment (ileahu-ellation) [inline src/App.js]: This file has a lot of changes, let's revert them

File: package.json (hunk 1 above)
Comment (arinacimpac) [reply package.json]: how do I do that exactly? when i access package.json this code isn't there

Comment (ileahu-ellation) [reply package.json]: hmm, maybe you're on the wrong branch?

**Review by ileahu-ellation — APPROVED** [review]

**Pull request merged**